src/pipecat/transports/services/daily.py

```diff
@@ -499,9 +499,11 @@ async def capture_participant_video(
         video_source: str = "camera",
         color_format: str = "RGB",
     ):
-        # Only enable camera subscription on this participant
+        # Try to enable camera and screen subscription on this participant
         await self.update_subscriptions(
-            participant_settings={participant_id: {"media": {"camera": "subscribed"}}}
+            participant_settings={
+                participant_id: {"media": {"camera": "subscribed", "screenVideo": "subscribed"}}
```

Comment:
Thank you! I believe this should be:

```
participant_settings={participant_id: {"media": {video_source: "subscribed"}}}
```

The settings from a previous call with a different video source will remain (e.g. if you first use `camera` and later use `screenVideo`). This way we only subscribe to what we need to.